### PR TITLE
Script for creating new env directories in flux v2

### PR DESCRIPTION
**Pull request by adusumillipraveen**
will do a follow up for preview

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: migration/create-environment.sh [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+mkdir -p $CLUSTER_DIRECTORY/base
+
+cp k8s/$ENVIRONMENT/pub-cert.pem  $CLUSTER_DIRECTORY/pub-cert.pem
+
+(
+cat <<EOF
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../../apps/flux-system/sbox/base
```
Comment (timja) [inline migration/create-environment.sh:23]: sbox?

File: migration/create-environment.sh [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+    substitute:
+      ENVIRONMENT: "$ENVIRONMENT"
+      CLUSTER: "01"
+      ENV_MONITOR_CHANNEL: "aks-monitor-$ENVIRONMENT"
+      KEYVAULT_ENVIRONMENT: "$ENVIRONMENT"
+EOF
+) > "$FLUX_SYSTEM_DIRECTORY/01/kustomize.yaml"
+
+
+kubectx cft-sbox-00-aks-admin
```
Comment (timja) [inline migration/create-environment.sh:113]: maybe should be a variable at the top for the cluster name? and verify that kubectx is installed?

**Review by timja — APPROVED** [review]
comments are non blocking

File: migration/create-environment.sh (hunk 2 above)
Comment (adusumillipraveen) [reply migration/create-environment.sh]: change it to just relogin.

File: migration/create-environment.sh [hunk 3]
```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env bash
+set -ex
+
+ENVIRONMENT=$1
+
+#Logging to cft sbox to pull secrets
+az aks get-credentials --resource-group cft-sbox-00-rg --name cft-sbox-00-aks --subscription b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb --overwrite-existing -a && kubectl config use-context cft-sbox-00-aks-admin
```
Comment (timja) [inline migration/create-environment.sh:7]: is the use-context needed? won't it swap you anyway?

Comment (adusumillipraveen) [reply migration/create-environment.sh]: never realised, always had it stored that way :)

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**